recipes/gmaps/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gmaps" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "1fa461c7544fb95310b34e1a9be0d1e27af7c805f64a206eb560e9cdaef857da" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - jupyter
+  run:
+    - python
+    - ipywidgets >=6.0.0
+    - geojson >=1.3.4
+
+test:
+  imports:
+    - gmaps
+
+about:
+  home: http://github.com/pbugnion/gmaps
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
Please add an issue/PR upstream to add the license file to the `MANIFEST.in` for future packages. That way we can reuse their license instead of having our own copy. 😉